content/integrations/kubernetes.md

```diff
@@ -5,9 +5,47 @@ kind: integration
 git_integration_title: kubernetes
 ---
 
+# Overview
+
 Get metrics from your Kubelets in real time to:
 
-* Visualize your Kubernetes cluster performance
```

Comment:
wait, you changed it from the correct spelling to a misspelling. It is 'visualize'